bin/start_backend.sh

```diff
@@ -185,6 +185,21 @@ if [[ $(ulimit -n) -lt 60000 ]]; then
 fi
 
 START_BE_CMD="${NUMA_CMD} ${STARROCKS_HOME}/lib/starrocks_be"
+
+# Add be cgroup hard limit.
+if [ -f ${STARROCKS_HOME}/conf/cgroup.conf ]; then
+    source ${STARROCKS_HOME}/conf/cgroup.conf
+    if [[ ${IF_USE_CPU_CGROUP} -eq 0 ]]; then
+        CGROUP_CPU_DIR=/sys/fs/cgroup/cpu/${CPU_CGROUP_NAME}/
```

Comment:
have to check if the system has `/sys/fs/cgroup` directory, whether the cgroup is enabled or not.